src/Controls/Maps/src/MapElement.cs

```diff
@@ -33,6 +34,7 @@ public float StrokeWidth
 		}
 
 		[EditorBrowsable(EditorBrowsableState.Never)]
-		public object MapElementId { get; set; }
+		public object? MapElementId { get; set; }
```

Comment:
This is part of a public interface - it doesn't need EditorBrowsable.Never.

Reply:
Done